README.md

```diff
@@ -123,7 +123,7 @@ To make further requests, check out the documentation - this gem should follow t
 ## Contributing to rubypress
 
 Pull requests welcome.
- 
+
```

Comment:
Hey man, this must be my atom text editor that's doing some automatic fixing of blank lines (it's the editor from Github, just an amazing piece of code). I didn't manually do anything, and if you look at it carefully, you will see that nothing really happens.